Sources/SPTDataLoaderServiceSessionSelector.m

```diff
@@ -27,6 +27,8 @@ Licensed to the Apache Software Foundation (ASF) under one
 @interface SPTDataLoaderServiceDefaultSessionSelector ()
 
 @property (nonatomic, strong, readonly) NSURLSessionConfiguration *configuration;
+@property (nonatomic, strong, readonly, nullable) NSURLSessionConfiguration *backgroundConfiguration;
```

Comment:
what makes the `backgroundConfiguration` special?
Cannot the `configuration` be used by just changing the needed fields?

Reply:
A background configuration object should be created using this method
https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/1407496-backgroundsessionconfigurationwi

Reply:
ok so can it be created internally, like it is done in `createWaitingSession`, instead of having it injected?

Reply:
Yeah, it can be created internally